src/site/content/en/fast/reduce-javascript-payloads-with-code-splitting/index.md

```diff
@@ -23,6 +25,18 @@ significantly. Instead of shipping all the JavaScript to your user as soon as
 the first page of your application is loaded, split your bundle into
 multiple pieces and only send what's necessary at the very beginning.
 
+## Why is code splitting beneficial?
+
+Code splitting is a technique that seeks to minimize startup time. When we ship less JavaScript at startup, we can get applications to be [interactive faster](/tti/) by minimizing main thread work during this critical period.
+
+When it comes to [Core Web Vitals](/vitals/), reducing JavaScript payloads downloaded at startup will contribute to better [First Input Delay (FID)](/fid/) and [Interaction to Next Paint (INP)](/inp/) times. The reasoning behind this is that, by freeing up the main thread, the application is able to respond to user inputs more quickly by reducing JavaScript parse, compile, and execution-related startup costs.
+
+Depending on your website's architecture&mdash;particularly if your website relies heavily on client-side rendering&mdash;reducing the size of JavaScript payloads responsible for rendering markup may lead to improved [Largest Contentful Paint (LCP)](/lcp/) times in some situations through reducing the [LCP candidate's render delay](/optimize-lcp/#2-eliminate-element-render-delay).
```

Comment:
@tunetheweb, how does this read?

Reply:
Bit confusing to be honest. I think there are two distinct things:
- General render-delay if main thread is too busy to process LCP image (particularly for SPAs).
- Discovery delay if the LCP depends on client-side code executing.

Reply:
I push a commit to revise this language a bit. Let me know how it scans.